bubblewrap.c

```diff
@@ -1664,6 +1666,18 @@ parse_args_recurse (int          *argcp,
         {
           print_version_and_exit ();
         }
+      else if (strcmp (arg, "--exec") == 0)
+        {
+          if (argc < 2)
+            die ("--exec takes one argument");
+
+          if (opt_exec_path != NULL)
+            warn_only_last_option ("--exec");
```

Comment:
New features should not use `warn_only_last_option()`: it was a hack to issue warnings about misleading command-lines, while still keeping backwards-compatible behaviour, for options that were historically allowed to be repeated.

For new features that don't make sense to be repeatable, like this one, it should be a fatal error to try to specify the option more than once.